kin/version.py

```diff
@@ -1 +1 @@
-__version__ = "2.0.0"
+__version__ = "2.1.0"
```

Comment:
do we have release notes? there's a release notes standard in github. maybe we should adopt it